Src/fahrenheit_to_celsius.py

```diff
@@ -0,0 +1,3 @@
+def fahrenheit_to_celsius(num):
+    temp = round((num - 32) * 5/9, 0)
```

Comment:
```suggestion
    temp = round((num - 32) * 5/9)
```
1. No need to add '0' as it is already being rounded.
2. A complete program with comments is good for others to understand.

Reply:
Thank you